classes/converter.php

```diff
@@ -25,6 +25,7 @@
 
 namespace tool_pdfpages;
 
+use Matrix\Exception;
```

Comment:
what is this for?

Reply:
Nothing, I accidentally imported it instead of `\Exception`, removing